mentions/src/main/proto/spine_examples/pingh/mentions/github_client_events.proto

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2024, TeamDev. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+syntax = "proto3";
+
+package spine_examples.pingh.mentions;
+
+import "spine/options.proto";
+
+option (type_url_prefix) = "type.pingh.spine.io";
+option java_package = "io.spine.examples.pingh.mentions.event";
+option java_outer_classname = "GitHubClientEventsProto";
+option java_multiple_files = true;
+
+import "spine_examples/pingh/mentions/identifiers.proto";
+import "spine_examples/pingh/mentions/mention.proto";
+import "spine_examples/pingh/github/github.proto";
+import "spine/net/url.proto";
+import "google/protobuf/timestamp.proto";
+
+// The GitHub token has been fetched.
+message GitHubTokenFetched {
+
+    // The ID of the GitHub client.
+    GitHubClientId id = 1;
+
+    // The new token.
+    spine_examples.pingh.github.GitHubToken token = 2 [(required) = true];
+}
+
+// The fetching of mentions from GitHub requested.
+message MentionsFetchingRequested {
```

Comment:
Why don't we stick exactly to what we have on the board?

![Screenshot 2024-05-10 at 18 27 33](https://github.com/spine-examples/Pingh/assets/82468/b5b0e7f7-3ea8-4de8-8572-610f13dba5f0)


The names here and the text there should be exactly the same thing.